src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -100,6 +104,8 @@ private String getHIUId(ConsentDetail consentDetail) {
                                                         String requestId,
                                                         List<GrantedConsent> grantedConsents) {
         String patientId = TokenUtils.readUserId(authorization);
```

Comment:
Could you please patient ID from controller to service?

Reply:
May be not for you @angshu, I will ask chethan to do that.

Reply:
done